go/v1beta1/storage/esutil/types.go

```diff
@@ -54,6 +54,7 @@ type EsSearch struct {
 	Sort     map[string]EsSortOrder `json:"sort,omitempty"`
 	Collapse *EsSearchCollapse      `json:"collapse,omitempty"`
 	Pit      *EsSearchPit           `json:"pit,omitempty"`
+	Routing  string
```

Comment:
`json:"routing,omitempty"`

Reply:
We actually want to ignore this field completely when serializing to JSON.  Maybe there's a better place to put this to make it more clear.

Reply:
To be more clear with this comment - if we wanted the routing value to be serialized to JSON to be sent to Elasticsearch, it would actually be `_routing` nested within something like a `term` query.  But I'm trying to use it more as a parameter to search rather than part of the underlying payload.

https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-routing-field.html